jsonforms-editor/src/text-editor/components/JsonEditorDialog.tsx

```diff
@@ -74,15 +76,37 @@ export const JsonEditorDialog: React.FC<JsonEditorDialogProps> = ({
   onCancel,
 }) => {
   const classes = useStyles();
-  const [content, setContent] = useState(initialContent);
+
+  const modelUri = Uri.parse('a://b/schema.json');
```

Comment:
does `a://b/` make sense? shouldn't we rather use something more speaking?

Reply:
I changed it `json://core/specification/...`